pandas/formats/style.py

```diff
@@ -444,15 +444,17 @@ def apply(self, func, axis=0, subset=None, **kwargs):
 
         Parameters
         ----------
-        func: function
-        axis: int, str or None
+        func : function
+            ``func`` should take a Series or DataFrame (depending
```

Comment:
And if we are doing nitpicky :-) The numpydoc standard says that references to keywords should have single backticks ...